remix-gql-tailwind/app/components/UserProfile/UserProfile.stories.tsx

```diff
@@ -3,8 +3,8 @@ import { Story, Meta } from '@storybook/react';
 import UserProfile from './UserProfile.view';
 
 const exampleData = {
-  avatarUrl: 'https://avatars.githubusercontent.com/u/2487968?v=4',
-  bio: 'Senior Software Engineer <a class="user-mention" data-hovercard-type="organization" data-hovercard-url="/orgs/thisdot/hovercard" href="https://github.com/thisdot">@thisdot</a>',
+  avatarUrl: '//avatars.githubusercontent.com/u/2487968?v=4',
+  bio: 'Senior Software Engineer <a class="user-mention" data-hovercard-type="organization" data-hovercard-url="/orgs/thisdot/hovercard" href="//github.com/thisdot">@thisdot</a>',
```

Comment:
This is passed as a string, and changing the URLs inside of it will cause problems when it is rendered as inner HTML.